README.md

```diff
@@ -53,46 +58,50 @@ Host *.stanford.edu
     GSSAPIDelegateCredentials yes
 ```
 
-### Problems with Authentication?
+### Set Dor-Services-App and Dor-Services Tokens
 
-If specs fail because they get through authentication without finding the "duo_iframe" "Send Me a Push", add `automatic_authentication: true` to `config/settings.local.yml`.
+Some integration tests use the `dor-services-client` to interact with the `dor-services-app`, others directly (or indirectly) use the `dor-services-app`.  In order to successfully use the dor-services-app API, you must first have a token set. (To generate dor-services-app tokens, see the [dor-services-app README](https://github.com/sul-dlss/dor-services-app#authentication).)
 
-You may also want to lower the time value of `post_authentication_text_timeout` in `config/settings.local.yml`.
+Place the value in `config/settings/{ENV}.local.yml`. See `config/settings.yml` for the expected YAML syntax. Note that you'll need to do this for each environment (currently: staging and qa), and you'll need top level keys of *both* `dor-services-app` and `dor-services` (same token). See `settings.yml` for explanation.
```

Comment:
The `dor_services_app` key is never used and should probably be removed.

Reply:
ooo!   I'll try it.